domenetjenester/dokumentbestiller/src/main/java/no/nav/foreldrepenger/dokumentbestiller/DokumentBestillerTjeneste.java

```diff
@@ -37,13 +40,12 @@ public void produserVedtaksbrev(BehandlingVedtak behandlingVedtak) {
 
         var behandling = behandlingRepository.hentBehandling(behandlingsresultat.getBehandlingId());
 
-        DokumentMalType dokumentMal;
         DokumentMalType opprinneligDokumentMal = null;
+        var dokumentMal = velgDokumentMalForVedtak(behandling, behandlingsresultat.getBehandlingResultatType(), behandlingVedtak.getVedtakResultatType(), behandlingVedtak.isBeslutningsvedtak(), klageRepository);
+
         if (Vedtaksbrev.FRITEKST.equals(behandlingsresultat.getVedtaksbrev())) {
+            opprinneligDokumentMal = erVedtakMedEndringIYtelse(behandlingsresultat) ? DokumentMalType.ENDRING_UTBETALING : dokumentMal;
```

Comment:
for lesbarhet; burde navnet være feks endretVedtakOgKunEndringIFordeling?

Reply:
selvsagt :)